modules/ROOT/pages/import/modeling.adoc

```diff
@@ -11,7 +11,12 @@ Relationships need to have a _type_ and one or more _properties_.
 You can add a new model and see your available models (if you have any) in the *Graph models* tab.
 If you add a new model, you can either either *Connect to a data source* to select a source from your list (see xref:import/file-provision.adoc[] for information about data sources), or you can drag and drop (or browse for) local flat files.
 
-Once you have defined your data source, you have three options to create your model:
+Once you have defined your data source, you are taken to the *data model panel*.
```

Comment:
```suggestion
Once you have defined your data source, you can choose to create a model from the data source, and it will take you to the modelling page*.
```